crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -83,47 +121,55 @@ impl Plugin for VisibilityPlugin {
     fn build(&self, app: &mut bevy_app::App) {
         use VisibilitySystems::*;
 
-        app.add_system_to_stage(
```

Comment:
Best viewed with "hide whitespace" - added a resource and system and it changed the indentation